process-automation/latest/model/connectors/teams.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Microsoft Teams connector
+---
+
+The Microsoft Teams connector uses the [Microsoft Graph](https://slack.com){:target="_blank"} API to intergrate with Microsfot Teams.
+
+The Teams connector is displayed on the process diagram with the Teams logo.
+
+> **Important**: The Teams connector and user both require a Microsoft Teams client. The Teams connector requires a Microsoft Teams account and is separate to the Alfresco hosted environment and should be created and managed by an administrator.
```

Comment:
I think you can split the last sentence so it's easier to read

Reply:
Adjusted